src/Pkg.jl

```diff
@@ -103,24 +103,32 @@ Add a package to the current project. This package will be available by using th
 a package, also inside that package.
 
 ## Resolution Tiers
-`Pkg` resolves the set of packages in your environment using a tiered algorithm.
-The `preserve` keyword argument allows you to key into a specific tier in the resolve algorithm.
-The following table describes the argument values for `preserve` (in order of strictness):
+`Pkg` resolves the set of packages in your environment using an algorithm that operates in one of four
+tiers, or modes. Which of these is used can be controlled by the `preserve` keyword argument,
+which should take one of the following values (in order of decreasing strictness):
 
 | Value             | Description                                                                         |
 |:------------------|:------------------------------------------------------------------------------------|
 | `PRESERVE_ALL`    | Preserve the state of all existing dependencies (including recursive dependencies)  |
-| `PRESERVE_DIRECT` | Preserve the state of all existing direct dependencies                              |
+| `PRESERVE_DIRECT` | Preserve the state of all existing direct dependencies. Error if this is impossible.|
```

Comment:
Doesn't this apply to all of them?

Reply:
It does, the others just lacked space. Maybe it should be elsewhere.

Reply:
Isn't this kind of implied?